setup.py

```diff
@@ -10,37 +9,42 @@
 from setuptools import setup, find_packages
 
 
-# Helpers
 def read(path):
+    """Read a text file at the given relative path."""
     basedir = os.path.dirname(__file__)
     return io.open(os.path.join(basedir, path), encoding='utf-8').read()
 
 
-# Prepare
-readme = read('README.md')
-license = read('LICENSE.txt')
-requirements = read('requirements.txt').split()
-requirements_dev = read('requirements.dev.txt').split()
-package = json.loads(read('package.json'))
+README = read('README.md')
+LICENSE = read('LICENSE.md')
+INFO = json.loads(read(os.path.join('tabulator', 'info.json')))
+INSTALL_REQUIRES = [
+    'six>=1.9',
+    'xlrd>=0.9',
+    'ijson>=2.0',
+    'chardet>=2.0',
+    'openpyxl>=2.0',
+    'jsontableschema>=0.5'
+]
+TESTS_REQUIRE = read('tests_require').split()
```

Comment:
I had a few problems with this pattern when the tests requirements file got a bit more complex. For example, if we add `-e git+...`, this will break. The format for the requirements in setup.py and requirements.txt seems to be different (except for the basic case).

At least, we should use `splitlines()` here.